The ports built correctly during the Azure Pipeline but failed with this error:

```
##[error]PowerShell wrote one or more lines to the standard error stream.
##[error]pwsh(72699) MallocStackLogging: can't turn off malloc stack logging because it was not enabled.

##[error]pwsh(72873) MallocStackLogging: can't turn off malloc stack logging because it was not enabled.

##[error]pwsh(72875) MallocStackLogging: can't turn off malloc stack logging because it was not enabled.
```

I don't think this is related to the `robotraconteur` package?

--- a/ports/robotraconteur/portfile.cmake
+++ b/ports/robotraconteur/portfile.cmake
@@ -15,7 +15,7 @@ vcpkg_cmake_configure(
 	OPTIONS
 	    -DBUILD_GEN=ON
 	    -DBUILD_TESTING=OFF
-		-DCMAKE_CXX_STANDARD=11
+	    -DCMAKE_CXX_STANDARD=11
 )
 
 vcpkg_cmake_install()

--- a/ports/robotraconteur/vcpkg.json
+++ b/ports/robotraconteur/vcpkg.json
@@ -4,7 +4,7 @@
   "description": "The Robot Raconteur communication framework core library",
   "homepage": "http://robotraconteur.com",
   "license": "Apache-2.0",
-  "supports": "(windows & (x86 | x64)) | (linux & (x86 | x64 | arm64 | arm)) | (osx & (x64 | arm64))",
+  "supports": "(windows & (x86 | x64)) | (linux & (x86 | x64 | arm64 | arm32)) | (osx & (x64 | arm64))",
   "dependencies": [
     "boost-algorithm",
     "boost-array",

--- a/versions/r-/robotraconteur.json
+++ b/versions/r-/robotraconteur.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "59fd72d178df2c7f291fa0397ea22e53c1a5eed1",
+      "git-tree": "5607a942906a086cd399770048006177c2371081",
       "version-semver": "1.2.0",
       "port-version": 0
     },

